docs/refs/configuring_mbed_os.md

```diff
@@ -1,13 +1,17 @@
 ### Configuring Arm Mbed OS
 
 #### Thread memory model
-All threads in mbed OS share a global heap. By default, memory for thread stacks is dynamically allocated from the global heap. User spawned thread stacks can be allocated from other memory areas, and this will be explored later in this document. The size of the global heap depends on the compiler. The heap size is static when compiling with IAR, and it is different for each target in mbed OS. It can be found in the IAR linker configuration files nested inside target directories. For example, the linker configuration file for a K66F can be found [here](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/device/TOOLCHAIN_IAR/MK66FN2M0xxx18.icf#L49-L51). For the GCC and ARM compilers, the heap size is dynamic based on RAM usage. 
+
+All threads in Mbed OS share a global heap. By default, memory for thread stacks is dynamically allocated from the global heap. User spawned thread stacks can be allocated from other memory areas. The size of the global heap depends on the compiler. The heap size is static when compiling with IAR, and it is different for each target in Mbed OS. You can find it in the IAR linker configuration files nested inside target directories. For example, you can find the linker configuration file for a K66F [here](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/device/TOOLCHAIN_IAR/MK66FN2M0xxx18.icf#L49-L51). For the GCC and ARM compilers, the heap size is dynamic based on RAM usage.
```

Comment:
- Query: Would it still be correct to change the second sentence to "By default, the global heap dynamically allocates memory for thread stacks." for active voice?
- Query: Would it be correct to change the third sentence to "You can allocate user spawned thread stacks from other memory areas." for active voice?